docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -184,6 +186,113 @@ export default StyledComponentsButton;
+
+```js
+import React from "react";
+import { css } from "emotion";
+import Button from "@material-ui/core/Button";
+
+const buttonStyles = css`
+  background: linear-gradient(45deg, #fe6b8b 30%, #ff8e53 90%);
+  border-radius: 3;
+  border: 0;
+  color: white;
+  height: 48px;
+  padding: 0 30px;
+  box-shadow: 0 3px 5px 2px rgba(255, 105, 135, 0.3);
+`;
+
+// We just assign them the Button's className attribute
+function EmotionButton() {
+  return (
+    <div>
+      <Button>Material-UI</Button>
+      <Button className={buttonStyles}>Emotion</Button>
+    </div>
+  );
+}
+export default EmotionButton
+```
+
+[![Edit Emotion](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/yw93kl7y0j)
+
+The styles created with **css()** can also be mapped to class names using the `classes` prop. This is useful when you want to customize the styles of deeper elements within a component.
+
+```js
+import React from "react";
+import { css } from "emotion";
+import Button from "@material-ui/core/Button";
+
+const styles = {
+  button: css({
+    background: "linear-gradient(45deg, #fe6b8b 30%, #ff8e53 90%)",
+    borderRadius: 3,
+    border: 0,
+    height: 48,
+    padding: "0 30px",
+    boxShadow: "0 3px 5px 2px rgba(255, 105, 135, 0.3)"
+  }),
+  label: css({
+    color: "white"
+  })
+};
+
+function EmotionButton() {
+  return (
+    <div>
+      <Button>Material-UI</Button>
+      <Button className={styles.button} classes={{ label: styles.label }}>
+        Emotion
+      </Button>
+    </div>
+  );
+}
+export default EmotionButton
+```
+
+[![Edit Material UI - Emotion Button 2](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/24om4jl05y)
+
+**Note:** By default Emotion and JSS both inject their styles at the bottom of the `<head>`. If you don't want to mark style attributes with **!important**, you need to change [the CSS injection order](/customization/css-in-js#css-injection-order), as in the examples.
+
+## React Emotion
+
+The **styled()** function can be used to customize the root styles of any Material-UI component.
+
+```js
+import React from "react";
+import styled from "react-emotion";
+import Button from "@material-ui/core/Button";
+
+const StyledButton = styled(Button)`
+  background: linear-gradient(45deg, #fe6b8b 30%, #ff8e53 90%);
+  border-radius: 3;
+  border: 0;
+  color: white;
+  height: 48px;
+  padding: 0 30px;
+  box-shadow: 0 3px 5px 2 px rgba(255, 105, 135, 0.3);
+`;
+
+function ReactEmotionButton() {
+  return (
+    <div>
+      <Button>Material-UI</Button>
+      <StyledButton>React Emotion</StyledButton>
+    </div>
+  );
+}
+
+export default ReactEmotionButton;
+```
+
+[![Edit Material UI - React Emotion Button ](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/nkx4p8rw9l)
```

Comment:
`Button ]`

Extra space?

Reply:
I'll fix that

Reply:
Also, should I change the titles of the sandbox demos to be more consistent with other examples?

```diff
-Material UI - React Emotion Button
+React Emotion Button
```